docs/site/DataSources.md

```diff
@@ -71,6 +71,10 @@ await bookDs.connect();
 app.dataSource(bookDs, dsName);
 ```
 
-To use the newly created datasource, call its `.connect()` method and attach it
-to the app using `app.dataSource()` method. Note, this method will be available
-only on `RepositoryMixin` apps.
+For details about datasource options, refer to the
+[DataSource documentation])(https://apidocs.strongloop.com/loopback-datasource-juggler/#datasource)
+.
+
+Attach the newly created datasource to the app by calling `app.dataSource()`.
+Note, the `app.repository()` method is available only on application classes
```

Comment:
```suggestion
Note, the `app.dataSource` method is available only on application classes
```

Have you considered using our `note.html` alert? https://loopback.io/doc/en/contrib/alerts.html

Reply:
Did for other instances too.